enterprise-suite/tests/e2e/cypress/integration/workload-page.spec.ts

```diff
@@ -17,7 +17,7 @@ describe('Workload Page Test', () => {
 
   it('validate control icons', () => {
     Util.validateControlIconContains('Grafana');
-    // Util.validateControlIconContains('Kubernetes');
+    //  Util.validateControlIconContains('Kubernetes');
```

Comment:
Doesn't look like anything changed?

Reply:
I mistakenly pushed a commit to master (removing this kubernetes icon test) then backed it out (via reset --hard 4ffccd1   <- my commit id) so this might be an artifact of that.

Reply:
Well master has this change already - see https://github.com/lightbend/helm-charts/blob/master/enterprise-suite/tests/e2e/cypress/integration/workload-page.spec.ts

Surprised you could push to master, thought we disabled that!